lomond/websocket.py

```diff
@@ -287,14 +302,16 @@ def build_request(self):
             "GET {} HTTP/1.1".format(self.resource).encode('utf-8')
         ]
         version = '{}'.format(constants.WS_VERSION)
-        headers = [
+        headers = self._headers[:]
```

Comment:
This maybe simplistic as the dev might attempt to set the automatically generated headers. It will suffice for now.